### PR TITLE
Doc: Adding cluster-proxy doc

### PR DESCRIPTION
**Pull request by yue9944882**
> Signed-off-by: yue9944882 <291271447@qq.com>

### CODE REVIEW

File: content/en/getting-started/integration/cluster-proxy.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+---
+title: Cluster proxy
+weight: 5
+---
+
+Cluster proxy is an OCM addon providing L4 network connectivity from hub
```
Comment (mikeshng) [inline content/en/getting-started/integration/cluster-proxy.md:6]:
```suggestion
[Cluster proxy](https://github.com/open-cluster-management-io/cluster-proxy) is an OCM addon providing L4 network connectivity from hub 
```

File: content/en/getting-started/integration/cluster-proxy.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+## Background
+
+The original architecture of OCM allows a cluster from anywhere to be
+registered and managed by OCM's control plane (i.e. the hub cluster)
+as long as the [klusterlet agent](https://open-cluster-management.io/getting-started/core/register-cluster/) 
+can reach hub cluster's endpoint. So the minimal requirement for the 
+managed cluster's network infrastructure in OCM is "klusterlet -> hub" 
+connectivity. However, there are still some cases where the components 
+in the hub cluster hope to proactively dail/request the services in the 
+managed clusters which will need the "hub -> kluster" connectivity on
```
Comment (mikeshng) [inline content/en/getting-started/integration/cluster-proxy.md:26]:
```suggestion
managed clusters which will need the "hub -> klusterlet" connectivity on 
```
Earlier there was a usage of `"klusterlet -> hub"`so maybe using `klusterlet` here is better.

File: content/en/getting-started/integration/cluster-proxy.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+* Ensure [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl) is installed.
+  
+* Ensure [`helm`](https://helm.sh/docs/intro/install/) is installed.
+
+## Installation
+
+To install the cluster proxy addon to the OCM control plane, run:
+
+```shell
+$ helm repo add ocm https://open-cluster-management.oss-us-west-1.aliyuncs.com
```
Comment (mikeshng) [inline content/en/getting-started/integration/cluster-proxy.md:103]: Side discussion: maybe we should create a helm chart repo for OCM and use https://github.com/helm/chart-releaser-action to manage it. That way the helm repo will be under ocm io github domain.

**Review by mikeshng — CHANGES_REQUESTED** [review]
This is a really well written article. Thank you Min. I just have some minor nits. I will defer to Qiu Jian for another look. 
I did notice you only added the `en` file. Maybe just copy and paste it to the `zh` folder as well because that's how other not yet translated pages are currently.

File: content/en/getting-started/integration/cluster-proxy.md (hunk 1 above)
Comment (yue9944882) [reply content/en/getting-started/integration/cluster-proxy.md]: updated

File: content/en/getting-started/integration/cluster-proxy.md (hunk 2 above)
Comment (yue9944882) [reply content/en/getting-started/integration/cluster-proxy.md]: rephrased to klusterlet

File: content/en/getting-started/integration/cluster-proxy.md (hunk 3 above)
Comment (yue9944882) [reply content/en/getting-started/integration/cluster-proxy.md:104]: i think so. we should probably start by creating a repo under the organization (e.g. named `chart`), is it? and i remember the azure folks are interested in hosting a OSS bucket for OCM charts. am fine w/ either ways tho.

Comment (ryanzhang-oss) [reply content/en/getting-started/integration/cluster-proxy.md:104]: Yes, we can definitely host the OCM helm charts. The most straightforward way is to just put it on a blob in Azure. However, I wonder if it's okay to expose the azure related URL in the helm repo info?

Comment (yue9944882) [reply content/en/getting-started/integration/cluster-proxy.md:104]: @ryanzhang-oss not at all. remember the official kubernetes images are hosted on `gcr.io`, it's fine to drop the release builds on any vendor as long as we can govern the release process.

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**